parachain/runtime/interlay/src/lib.rs

```diff
@@ -556,8 +556,8 @@ impl pallet_multisig::Config for Runtime {
 
 parameter_types! {
     pub const ProposalBond: Permill = Permill::from_percent(5);
-    pub ProposalBondMinimum: Balance = 5;
-    pub ProposalBondMaximum: Option<Balance> = None;
+    pub ProposalBondMinimum: Balance = 5 * UNITS;
+    pub ProposalBondMaximum: Balance = 25 * UNITS;
```

Comment:
How did you decide on this value of 25 kint?

Reply:
Taken from Karura: https://github.com/AcalaNetwork/Acala/blob/72316317eed96a8bd7dde55873bc88a8945557af/runtime/karura/src/lib.rs#L582-L583

Reply:
Requested dom's approval for this. Just so we're clear: to do a treasury proposal, you have to lock `min(ProposalBondMaximum, 0.05* spending_amount)`

Reply:
`min(ProposalBondMinimum, 0.05 * spending_amount)`

Reply:
> `min(ProposalBondMinimum, 0.05 * spending_amount)`

I don't think that's right. If you want to include the Minimum in the calculation, I think it's  `max(ProposalBondMinimum, min(ProposalBondMaximum, 0.05* spending_amount))`